### PR TITLE
Update docs on fresher's selection

### PR DESCRIPTION
**Pull request by proffapt**
**Write path(s) which will be affected by this Pull Request.**
Example:
- `/events/freshers-selection`

**Why do you think it should be documented?**

- Archival of past 2 years FB post.
- Mentioning of `[mailing-scripts](https://github.com/kossiitkgp/mailing-scripts)` for ease in mailing

### CODE REVIEW

File: events/freshers-selection.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ Core parameters for hiring are
 
 The parameters are given equal weight.
 
+### Mailing
+
+All the mails sent during the selection process are sent via [mailing-scripts](https://github.com/kossiitkgp/mailing-scripts). It already has all of the templates of the mail which are to be sent during any round.
```
Comment (harshkhandeparkar) [inline events/freshers-selection.md:17]:
```suggestion
All the mails sent during the selection process are sent using the [mailing-scripts](https://github.com/kossiitkgp/mailing-scripts). It already has all of the templates of the mail which are to be sent during any round.
```

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]
Small change, otherwise LGTM.

File: events/freshers-selection.md (hunk 1 above)
Comment (proffapt) [reply events/freshers-selection.md:17]: What difference does it make (*￣▽￣)b

**Review by harshkhandeparkar — APPROVED** [review]
LGTM

**Review by rajivharlalka — APPROVED** [review]

**Review by chirag-ghosh — APPROVED** [review]

**Comment by proffapt** [comment]
So, can it be merged now?

**Pull request merged**